protocol.html

```diff
@@ -779,6 +786,7 @@ <h3 property="schema:name" content="Subscription Service Data Model">Subscriptio
                   <ul>
                     <li id="notify-subscription-id">One <code>id</code> property to identify the subscription service.</li>
                     <li id="notify-channelType">One <code>channelType</code> property to state the <a href="#notification-channel-types" rel="rdfs:seeAlso">notification channel type</a>.</li>
+                    <li id="notify-channelAuthScheme">One or many <code>channelAuthScheme</code> property to state the supported authentication schemes.</li>
```

Comment:
I couldn't suggest a change to add a paragraph/section (perhaps right after #authentication-authorization or a new subsection of the #protocol section.) The following suggestion would refer to Authentication Scheme.

```suggestion
                    <li id="notify-channelAuthScheme">One or many <code>channelAuthScheme</code> property to state the supported <a href="#authentication-scheme">authentication schemes</a>.</li>
```